keyboards/annepro2/protocol.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * (c) 2021 by Tomasz bla Fortuna
+ * License: GPLv2
```

Comment:
I don't enjoy copying two files between projects but git submodule for those 2 is rather even clumsier. I've got no better idea.

Tell me if you don't like the header too.

Reply:
I see. I guess we can add a comment here at the top saying that please keep the two files in sync. (I know you already have a line that say it is shared. I think it is better to be clear)